shell/app/node_main.cc

```diff
@@ -62,40 +63,26 @@ int SetNodeCliFlags() {
       "--use-openssl-ca",
   });
 
-  const auto argv = base::CommandLine::ForCurrentProcess()->argv();
-  std::vector<std::string> args;
-
   // TODO(codebytere): We need to set the first entry in args to the
   // process name owing to src/node_options-inl.h#L286-L290 but this is
   // redundant and so should be refactored upstream.
-  args.reserve(argv.size() + 1);
-  args.emplace_back("electron");
-
+  std::vector<std::string> args;
   for (const auto& arg : argv) {
-#if BUILDFLAG(IS_WIN)
-    const auto& option = base::WideToUTF8(arg);
-#else
-    const auto& option = arg;
-#endif
-    const auto stripped = base::StringPiece(option).substr(0, option.find('='));
+    const auto stripped = base::StringPiece(arg).substr(0, arg.find('='));
     if (disallowed.contains(stripped)) {
       LOG(ERROR) << "The Node.js cli flag " << stripped
                  << " is not supported in Electron";
       // Node.js returns 9 from ProcessGlobalArgs for any errors encountered
       // when setting up cli flags and env vars. Since we're outlawing these
-      // flags (making them errors) return 9 here for consistency.
-      return 9;
+      // flags (making them errors) exit with the same error code for
+      // consistency.
+      exit(9);
     } else {
-      args.push_back(option);
+      args.push_back(arg);
     }
```

Comment:
_(nit)_ Not new to this PR, but the `else {` and `}` lines 80 and 82 are not necessary & can be removed